modules/gapi/samples/pipeline_modeling_tool/pipeline.hpp

```diff
@@ -1,25 +1,38 @@
 #ifndef OPENCV_GAPI_PIPELINE_MODELING_TOOL_PIPELINE_HPP
 #define OPENCV_GAPI_PIPELINE_MODELING_TOOL_PIPELINE_HPP
 
+#include <iomanip>
+
 struct PerfReport {
-    std::string               name;
-    double  avg_latency       = 0.0;
-    double  throughput        = 0.0;
-    int64_t first_run_latency = 0;
-    int64_t elapsed           = 0;
-    int64_t compilation_time  = 0;
+    std::string name;
+    double  avg_latency        = 0.0;
+    int64_t min_latency        = 0;
+    int64_t max_latency        = 0;
+    int64_t first_latency      = 0;
+    double  throughput         = 0.0;
+    int64_t elapsed            = 0;
+    int64_t warmup_time        = 0;
+    int64_t num_late_frames    = 0;
     std::vector<int64_t> latencies;
 
     std::string toStr(bool expanded = false) const;
 };
 
 std::string PerfReport::toStr(bool expand) const {
     std::stringstream ss;
-    ss << name << ": Compilation time: " << compilation_time << " ms; "
-       << "Average latency: " << avg_latency << " ms; Throughput: "
-       << throughput << " FPS; First latency: "
-       << first_run_latency << " ms";
-
+    ss << name << ": "<< std::endl
+       << std::string(2, ' ') << "Warm up time: " << warmup_time << " ms" << std::endl
```

Comment:
`Compilation` -> `Warm up` because it might be confused with `OpenVINO` model compilation.